WIP-CHANGELOG.md

```diff
@@ -17,4 +17,17 @@
 
 ### Added
 
+
+- Added fluentd metrics
+- Enabled automatic compaction (cleanup) of pos_files for fluentd
+- Added and enabled by default an option for Grafana Viewers to temporarily edit dashboards and panels without saving.
+- New Prometheus rules have been added to forewarn against when memory and disk (PVC and host disk) capacity overloads
+- Added the possibility to whitelist IP addresses to the loadbalancer service
+- Added pwgen and htpasswd as requirements
+- Added the blackbox installation in the wc cluster based on ADR to monitor the uptime of internal services as well in wc .
+- Added option to enable index per namespace feature in fluentd and elasticsearch
+- Added optional off-site backup replication between two providers or regions using rclone sync
+- Added Prometheus alerts for the 'backup status' and 'daily checks' dashboards. Also, 's3BucketPercentLimit' and 's3BucketSizeQuotaGB' parameters to set what limits the s3 rule including will alert off.
```

Comment:
It looks like you have added too much here. The only addition should be the last line (the one you wrote for this PR)